src/js/tech/html5.js

```diff
@@ -70,6 +72,13 @@ class Html5 extends Tech {
       }
     }
 
+    if (this.featuresNativeVideoTracks) {
+      this.handleVideoTrackChange_ = Fn.bind(this, this.handleVideoTrackChange);
```

Comment:
I would be wary to change how it works for the text tracks themselves but OK with not doing it for the new video and audio track support.